Bruin Bite/Network/UserManager.swift

```diff
@@ -144,7 +144,7 @@ class UserManager {
     }
 
     func getNewAccessToken(refreshToken: String) {
-        self.provider.request(.refreshToken(refresh_token: refreshToken)) { result in
+        self.provider.request(.refreshToken(grant_type: "refresh_token", client_id: CLIENTID, client_secret: CLIENTSECRET, refresh_token: refreshToken)) { result in
```

Comment:
@sjl1826 can you point me to the docs that say why this will work?

Reply:
What will work? Like why do we need to provide grant_type for refresh_token? https://auth0.com/docs/tokens/refresh-token/current